### ISSUE TITLE
how to delpoy a model in hdfs

### ISSUE THREAD

**Issue by Lihengwannafly** [tobegit3hub/simple_tensorflow_serving#49]

**Comment by tobegit3hub** [comment]
Thanks for reporting. You can pass the model path as the specified HDFS path.

We may add more detailed document about this soon.

**Comment by Lihengwannafly** [comment]
> Thanks for reporting. You can pass the model path as the specified HDFS path.
> 
> We may add more detailed document about this soon.

simple_tensorflow_serving --model_base_path="hdfs://GZGD/171587/1/model/NeuralCF/base-model-output" --model_platform="tensorflow"
I deploy the above command,  but this error occurs: `tensorflow.python.framework.errors_impl.NotFoundError: Could not find directory`.

**Comment by tobegit3hub** [comment]
Could you please post the files in this HDFS directory? Try with `hadoop dfs -ls hdfs://GZGD/171587/1/model/NeuralCF/base-model-output`.

We have to make sure that the output path is the SavedModel files with the model version.

**Comment by Lihengwannafly** [comment]
> Could you please post the files in this HDFS directory? Try with `hadoop dfs -ls hdfs://GZGD/171587/1/model/NeuralCF/base-model-output`.
> 
> We have to make sure that the output path is the SavedModel files with the model version.

![image](https://user-images.githubusercontent.com/24928240/56875197-c7f3c080-6a71-11e9-8e48-bd1720d0b151.png)
the image is the screenshots of the hdfs path

**Comment by tobegit3hub** [comment]
So the `1551684206` is the model version, right? And are there `saved_model.pb` and `variables` in these directories of the versions?

**Comment by Lihengwannafly** [comment]
> So the `1551684206` is the model version, right? And are there `saved_model.pb` and `variables` in these directories of the versions?



> So the `1551684206` is the model version, right? And are there `saved_model.pb` and `variables` in these directories of the versions?

yes, of course. By the way, I used pip to install the tool, should I switch to use docker to install latest-hdfs.
![image](https://user-images.githubusercontent.com/24928240/56877038-e495f580-6a7d-11e9-9f7a-d5e0314e6c50.png)

**Comment by Lihengwannafly** [comment]
> > So the `1551684206` is the model version, right? And are there `saved_model.pb` and `variables` in these directories of the versions?
> 
> > So the `1551684206` is the model version, right? And are there `saved_model.pb` and `variables` in these directories of the versions?
> 
> yes, of course. By the way, I used pip to install the tool, should I switch to use docker to install latest-hdfs.
> ![image](https://user-images.githubusercontent.com/24928240/56877038-e495f580-6a7d-11e9-9f7a-d5e0314e6c50.png)



> So the `1551684206` is the model version, right? And are there `saved_model.pb` and `variables` in these directories of the versions?
yes, of course.
![image](https://user-images.githubusercontent.com/24928240/56877096-3a6a9d80-6a7e-11e9-9f31-354a2d6fb48b.png)
Should I use docker to install latest-hdfs?

**Comment by Lihengwannafly** [comment]
> So the `1551684206` is the model version, right? And are there `saved_model.pb` and `variables` in these directories of the versions?

yes, of course. By the way, I used pip to install the tool, should I switch to use docker to install latest-hdfs.
![image](https://user-images.githubusercontent.com/24928240/56877132-76056780-6a7e-11e9-84bd-37880c9daf49.png)
